### DISCUSSION TITLE
如何查看hexo版本

### DISCUSSION THREAD

**Discussion by zhang-jc** [hexojs/hexo#5587] [Question / Answer]
使用命令 hexo version只显示 hexo-cli 版本。但官方文档提供的是 hexo 与 nodejs 版本的对应关系，那如何查看安装的hexo版本？

**Comment by mrzzoxo** [comment]
``` shell
npm list
```
这个指令 可以看到

**Comment by uiolee** [comment]
![image](https://github.com/user-attachments/assets/d7f923be-b840-4001-b433-0f4025a6c75a)
